package.build.ts

```diff
@@ -361,6 +361,11 @@ export const pkg = (modules: Builder.ParsedModule[]) => ({
               title: {
                 type: "string",
               },
+              menu_type: {
```

Comment:
Please use `camelCase`.
```suggestion
              menuType: {
```

Reply:
Also please add some documentation to explain the difference between "hotkey" and "palette". I think the names are appropriate, but IMO it's still not obvious what the difference is.

Reply:
Some alternative naming (may not be better): "hotkey" vs. "fuzzy".

Reply:
not a fan of fuzzy as it suggests non-exact matching, but maybe there are still improvements on 'hotkey' :)